modules/services/preferences/src/main/java/au/com/shiftyjelly/pocketcasts/preferences/Settings.kt

```diff
@@ -62,7 +62,7 @@ interface Settings {
 
         const val CHROME_CAST_APP_ID = "2FA4D21B"
 
-        const val WHATS_NEW_VERSION_CODE = 9121
+        const val WHATS_NEW_VERSION_CODE = 9256
```

Comment:
Please, double check if this is the expected version code 🙏

Reply:
This isn't related to app version code. It is a separate code. It should be incremented normally